### PR TITLE
[ENG-23] Serialize meta for preprint findRecord

### PR DESCRIPTION
**Pull request by futa-ikeda**
## Purpose
- Allow a preprint's metrics metadata to be serialized in the `apiMeta` property

## Summary of Changes/Side Effects
- Normalize `findRecord` responses to include the response.meta into an `apiMeta` attribute so it can be used later in the app (specifically preprint's views/download metrics)

## Testing Notes
- this change is additive and doesn't change any existing user-facing behavior

## Ticket

https://openscience.atlassian.net/browse/ENG-23

## Notes for Reviewer
- I've opted to store the response.meta object into the `apiMeta` attribute as this seemed like the easiest way to access the impact metrics for a given preprint. I looked at using store.findMetadataFor('preprint'), but it seems this is deprecated.

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
         key: cached_node_modules_${{ hashFiles('**/yarn.lock') }}
         restore-keys: cached_node_modules_
     - if: steps.cache-node-modules.outputs.cache-hit != 'true'
-      run: yarn --prefer-offline --frozen-lockfile --ignore-engines
+      run: yarn --prefer-offline --ignore-engines
```
Comment (brianjgeiger) [inline .github/workflows/CI.yml:29]: Don't we prefer frozen lockfiles? What prompted the change?

Comment (brianjgeiger) [reply .github/workflows/CI.yml:29]: But you added a new lock file to this PR. Shouldn't that have all the updates it needs?

Comment (futa-ikeda) [reply .github/workflows/CI.yml:29]: Ahh, I didn't push the updated `yarn.lock` after moving faker... Will rectify

Comment (futa-ikeda) [reply .github/workflows/CI.yml]: realized that a lot of the yarn.lock file changes came from when I updated ember-faker, which I didn't need to do, so force-pushed without that commit in there to keep things a bit easier to read

**Review by brianjgeiger — DISMISSED** [review]

**Review by brianjgeiger — APPROVED** [review]

File: addon/serializers/osf-serializer.js [hunk 2]
```diff
@@ -141,5 +141,13 @@ export default DS.JSONAPISerializer.extend({
             documentHash.meta.total_pages = Math.ceil(documentHash.meta.total / documentHash.meta.per_page);
         }
         return documentHash;
+    },
+
+    normalizeSingleResponse(_, __, ___, ____, requestType) {
+        const documentHash = this._super(...arguments);
+        if (requestType === 'findRecord' && documentHash.meta) {
+            documentHash.data.attributes.apiMeta = documentHash.meta;
```
Comment (adlius) [inline addon/serializers/osf-serializer.js:149]: can we do `documentHash.data.attributes.meta = documentHash.meta` without adding `apiMeta: DS.attr()` to the model? In fact, I'd expect every resource to have a `meta` property and am surprised that there isn't.

**Pull request merged**